CMakeLists.txt

```diff
@@ -41,7 +45,7 @@ add_custom_command(OUTPUT ${pythonbindingsfile}
 				   COMMAND "${CMAKE_C_COMPILER}" -E ${LLVM_DEFINITIONS} -isystem ${LLVM_INCLUDE_DIRS} "${LLVM_INCLUDE_DIRS}/llvm-c/Core.h" | python "${CMAKE_SOURCE_DIR}/fcd/python/bindings.py" > ${pythonbindingsfile})
 
 ### header search paths file ###
-execute_process(COMMAND "${CMAKE_CXX_COMPILER}" -E -x c++ -v /dev/null OUTPUT_VARIABLE dummy ERROR_VARIABLE defaultHeaderSearchPathList)
```

Comment:
-std=gnu++14 is not necessary, this command is preprocessing /dev/null to get the default include path from the stderr output. In fact, it almost certainly should be invoking `${CMAKE_C_COMPILER}`, because right now it gets the C++ include path and fcd wouldn't know what to do with it.